app/graphql/types/query_type.rb

```diff
@@ -0,0 +1,20 @@
+module Types
+  class QueryType < Types::BaseObject
+    # Add root-level fields here.
+    # They will be entry points for queries on your schema.
+
+    # How to define query types: 
+    # field <:query_name>, resolver: Queries::ClassName(defined in query_file.rb)
+    # resolver: executes the logic in the resolve method of the above class 
+
+    # TODO: remove me
```

Comment:
We can remove the test_field at this point.

Reply:
DELETED